migrations/20200907080331-add-comment-filed-in-repositories-table.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return Promise.all([
+      queryInterface.addColumn("repositories", "comment", {
+        type: Sequelize.STRING(50),
```

Comment:
why comment size is 50? Comment can be more than 50 char

Reply:
i have changed to Text